client/packages/common/src/ui/layout/tables/material-react-table/useSimpleMaterialTable.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { MRT_RowData, MRT_ShowHideColumnsButton } from 'material-react-table';
+import { BaseTableConfig, useBaseMaterialTable } from './useBaseMaterialTable';
+import { Box } from '@mui/material';
+
+interface SimpleTableConfig<T extends MRT_RowData> extends BaseTableConfig<T> {
+  bottomToolbarContent?: string | React.JSX.Element | React.JSX.Element[];
+}
+
+export const useSimpleMaterialTable = <T extends MRT_RowData>({
+  bottomToolbarContent,
+  ...tableOptions
+}: SimpleTableConfig<T>) => {
+  const table = useBaseMaterialTable<T>({
```

Comment:
hm, i wonder if my sort/filter changes will muck this up? The sort/filtering of the underlying table on the detail view would also be getting applied to this one 👀 

Let's see after merging one of these, might be better to use a separate definition here, and just pull in the needed hooks (probably just our `useMaterialTableColumns` and maybe the styles stuff?)